### ISSUE TITLE
Arithmetic predicates for number (Integer or Float)

### ISSUE THREAD

**Issue by fabricereix** [Orange-OpenSource/hurl#95]
Predicates with numbers such as (equals, greaterThan, ...) are "arithmetic".

For example:
for the following json response
```
{
    "count": 5,
    "total": 50.0
}

```
All the following asserts are valid
```
jsonpath "$.count" equals 5
jsonpath "$.count" equals 5.0
jsonpath "$.count" greaterThan 1
jsonpath "$.count" greaterThan 1.0
jsonpath "$.total" equals 50
jsonpath "$.total" equals 50.0
jsonpath "$.total" greaterThan 0.2
jsonpath "$.total" greaterThan 0
```

The value type can be checked with an additional predicate
```
jsonpath "$.count" isInteger
jsonpath "$.total" isFloat
```

Additional:

The intent for the following assert might not be clear:
`jsonpath "$.count" equals 5.0`
Do we expect a float value?

In this case, Hurl could emit a warning like:
"You expected value should be replaced by an integer value"

**Comment by lepapareil** [comment]
👍

**Comment by fabricereix** [comment]
implemented in #99
